auth/service.go

```diff
@@ -730,15 +730,17 @@ func (svc service) UnassignUsers(ctx context.Context, token, id string, userIds
 		return err
 	}
 
-	if err := svc.Authorize(ctx, PolicyReq{
-		Subject:     token,
-		SubjectType: UserType,
-		SubjectKind: TokenKind,
-		Object:      id,
-		ObjectType:  DomainType,
-		Permission:  SwitchToPermission(relation),
-	}); err != nil {
-		return err
+	if relation != "" {
+		if err := svc.Authorize(ctx, PolicyReq{
+			Subject:     token,
+			SubjectType: UserType,
+			SubjectKind: TokenKind,
+			Object:      id,
+			ObjectType:  DomainType,
+			Permission:  SwitchToPermission(relation),
+		}); err != nil {
+			return err
+		}
 	}
 
```

Comment:
We should not allow editor to remove administrator. 
This check will be bypassed if relation is empty. 
We need to think this approch in different way